src/sentry/buffer/redis.py

```diff
@@ -73,6 +73,7 @@ def callback(self, buffer_hook_event: BufferHookEvent, data: RedisBuffer) -> boo
         try:
             callback = self._registry[buffer_hook_event]
         except KeyError:
+            logger.info("buffer_hook_event.missing", extra={"key_name": buffer_hook_event.value})
```

Comment:
nit: might be nice to make this an error log instead of an info log, and pass the exception.

Reply:
👍 I'll change it in a separate PR, really wanna get this out to do testing